src/main/java/Duke.java

```diff
@@ -1,10 +1,66 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String HORIZONTAL_LINES = "____________________________________________________________\n";
+    public static final String INDENTATIONS = "    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(INDENTATIONS + HORIZONTAL_LINES +
+                INDENTATIONS + "Hello! I'm Duke\n" +
+                INDENTATIONS + "What can I do for you?\n" +
+                INDENTATIONS + HORIZONTAL_LINES);
+        Scanner in = new Scanner(System.in);
+        String readLine;
+        Task[] tasks = new Task[100];
+        int taskIndex = 0;
+        while(true) {
```

Comment:
```suggestion
        while (true) {
```

I noticed similar cases in many other lines as well, it might be good to modify them to fully follow the style guide.